modules/protocols/snap/src/services/fldt/fldt-client.c

```diff
@@ -697,11 +697,11 @@ vs_fldt_client_add_file_type(const vs_update_file_type_t *file_type, vs_update_i
 
 /******************************************************************/
 static vs_status_e
-_fldt_destroy_client(struct vs_snap_service_t *service) {
+_fldt_destroy_client(vs_snap_service_user_data_t service_user_data) {
     uint32_t id;
     vs_fldt_client_file_type_mapping_t *file_type_mapping = _client_file_type_mapping;
 
-    (void)service;
+    (void)service_user_data;
```

Comment:
These functions are not a default implementations and they are not use service_user_data. Why do we add this excess parameter?